### PR TITLE
Issue 4343

### PR DESCRIPTION
**Pull request by ryg-git**
### Pull Request Checklist
Related to #4343 

- [x] Changes has been tested on an Android device or Android emulator with API 21
- [x] UI change has been tested on both light and dark themes
- [x] Accessibility has been taken into account. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#accessibility
- [x] Pull request is based on the develop branch
- [x] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#changelog
- [ ] Pull request includes screenshots or videos if containing UI changes
- [x] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [x] You've made a self review of your PR
- [ ] If you have modified the screen flow, or added new screens to the application, you have updated the test [UiAllScreensSanityTest.allScreensTest()](https://github.com/vector-im/element-android/blob/main/vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt#L73)

Signed-off-by: Yashawant <ygretharekar@outlook.com>

### CODE REVIEW

File: vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt [hunk 1]
```diff
@@ -1055,8 +1055,11 @@ class RoomDetailFragment @Inject constructor(
         val data = event.buildImageContentRendererData(dimensionConverter.dpToPx(66))
         val isImageVisible = if (data != null) {
             imageContentRenderer.render(data, ImageContentRenderer.Mode.THUMBNAIL, views.composerLayout.views.composerRelatedMessageImage)
+            views.composerLayout.views.composerRelatedMessageImage.isVisible = true
             true
         } else {
+            imageContentRenderer.clear(views.composerLayout.views.composerRelatedMessageImage)
+            views.composerLayout.views.composerRelatedMessageImage.isVisible = false
```
Comment (bmarty) [inline vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt:1062]: Maybe simplify the code duplication by moving this line outside of the if and else block and just write:
```kotlin
views.composerLayout.views.composerRelatedMessageImage.isVisible = isImageVisible
```

**Review by bmarty — COMMENTED** [review]
Thanks for the PR, it LGTM (I can also reproduce the original issue)
Just one small remark on the form.
Also the changelog file should ideally stay simple (one line). What about `Fix a display issue in the composer when the replied message is changed.`

**Comment by ouchadam** [comment]
> Thanks for the PR, it LGTM (I can also reproduce the original issue) Just one small remark on the form. Also the changelog file should ideally stay simple (one line). What about `Fix a display issue in the composer when the replied message is changed.`

I'm a big fan of having the PR description be the place for the detailed commentary

**Review by ouchadam — APPROVED** [review]
LGTM :100:  (assuming @bmarty remarks are actioned)

File: vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt (hunk 1 above)
Comment (ryg-git) [reply vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailFragment.kt:1062]: Yes that does make a lot of sense, I will update the code, thank you for the review!

**Comment by ryg-git** [comment]
> Thanks for the PR, it LGTM (I can also reproduce the original issue) Just one small remark on the form. Also the changelog file should ideally stay simple (one line). What about `Fix a display issue in the composer when the replied message is changed.`

Thank you for the reply, I will update the changelog file and move the description in the PR description box.

**Review by bmarty — APPROVED** [review]
Thanks for the update!

**Pull request merged**